tldw_Server_API/app/core/AuthNZ/create_admin.py

```
@@ -0,0 +1,160 @@
+        try:
+            from tldw_Server_API.app.core.AuthNZ.api_key_manager import get_api_key_manager
+
+            api_manager = await get_api_key_manager()
+            api_key_result = await api_manager.create_api_key(
+                user_id=user_id,
+                name="Initial Admin API Key",
+                description="Auto-generated during Docker bootstrap",
+                scope="admin",
+                expires_in_days=365,
+            )
+            print(f"[create-admin] Admin API key created (expires in 365 days).")
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)